com.github.taiko2k.avvie.yml

```diff
@@ -0,0 +1,48 @@
+app-id: com.github.taiko2k.avvie
+runtime: org.gnome.Sdk
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: avvie
+finish-args:
+- --share=ipc
+- --socket=fallback-x11
+- --device=dri
+- --socket=wayland
+build-options:
+  build-args:
+  - --share=network
+  env: {}
```

Comment:
FYI this section of the Flatpak manifest is unnecessary as the python packages are pre-downloaded by flatpak-builder (and checked against the checksum). Adding this will cause the build to fail when built in sandbox mode (i.e build by flathub)

I've committed a fix to master.